test/lib/browsers/playwright.ts

```diff
@@ -69,17 +65,13 @@ export class Playwright extends BrowserInterface {
         sources: true,
       })
       this.activeTrace = encodeURIComponent(url)
-
-      page.on('close', async () => {
-        await teardown(this.teardownTracing.bind(this))
-      })
```

Comment:
This unfortunately does not work as you have to stop tracing before the page gets closed